io_port.c

```diff
@@ -44,10 +44,22 @@
 #include <handler.h>
 
 #include "io_port.h"
+#include "ppi.h"
```

Comment:
This should be `#include <ppi.h>` and placed above by handler.h since it is from the include path rather than a relative path.